addons/survey/tests/common.py

```diff
@@ -167,7 +167,7 @@ def _access_submit(self, survey, token, post_data):
         return self.opener.post(url=url, json={'params': post_data})
 
     def _find_csrf_token(self, text):
-        csrf_token_re = re.compile("(input.+csrf_token.+value=\")([_a-zA-Z0-9]{51})", re.MULTILINE)
+        csrf_token_re = re.compile("(input.+csrf_token.+value=\")([_a-f0-9o]{40,})", re.MULTILINE)
```

Comment:
```suggestion
        csrf_token_re = re.compile("(input.+csrf_token.+value=\")([_a-f0-9]{40,}o)", re.MULTILINE)
```
could even add the optional timestamp back. Not sure why ther's a `_` in the pattern though?

Reply:
good point, though I'm not sure what is the value of such strict matching for basic testing purposes